website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Configuring Notifications"
+---
+
+### Overview
+
+Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+
+### Email
+
+To set up email noticiations first navigate to your **Profile** (found in the top right hand side of dbt Cloud) and the select **Notifications**.
```

Comment:
You can also go to Account Settings to set up notification for your entire team. Did you want to also provide this path?

Reply:
Thanks Amy! Just made a commit to include that aspect.

Reply:
@amychen1776 if you have a moment to review this change would be awesome :)